zktest/src/archive/test2/config.properties

```diff
@@ -2901,6 +2901,8 @@ F85-ZK-3824.zul=A,E,AuWriters,AuResponse,CONTENT_TYPE
 ##ztl##F85-ZK-3704.zul=A,E,NumberInputElement,FormatWidget,Intl
 ##zats##F85-ZK-3846.zul=A,E,Datebox,SimpleDateConstraint,format,separator
 F85-ZK-3852.zul=A,E,Chosenbox,inplace
+##ztl##B85-ZK-3945.zul=A,E,zindex,popup,window
+##zats##B85-ZK-3811.zul=A,E,PathELResolver,BindELResolver,getValue,tieValue

```

Comment:
1. wrong place
2. B85-ZK-3945.zul shouldn't be here

Reply:
ok sorry